src/components/calcite-tree/calcite-tree.tsx

```diff
@@ -108,7 +108,7 @@ export class CalciteTree {
       event.relatedTarget === this.el || !this.el.contains(event.relatedTarget as HTMLElement);
 
     if (focusedFromRootOrOutsideTree) {
-      this.el.tabIndex = -1;
+      this.el.removeAttribute("tabindex");
```

Comment:
For maintainability, can you add a comment about why the attribute is modified instead of the property?